CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog for Mantarray Frontend Components
 ===========================================
 
+0.6.1 (2022-02-03)
+------------------
+
+- Added expected firmware update duration to spinner widget
+- Added firmware update timeout message
+- Added minor styling updates
+- Changed subprotocol edit from Shift+Click to Double Click
```

Comment:
Can we also add additional controls hover modals?

Reply:
Yeah that's my bad, forgot to add that